Updating wording for contacting support (#1045)
codatio/codat-docs: docs/get-started/migration.md

:::success Plan your migration

> Every migration is different so please make sure you talk to your solutions engineer, or raise a ticket with our support team through [this form](https://codat.zendesk.com/hc/en-gb/requests/new) as early as possible to discuss and plan your migration.

Reviewer: ```suggestion
Every migration is different, so make sure you talk to your solutions engineer or use our [support request form](https://codat.zendesk.com/hc/en-gb/requests/new) as early as possible to discuss and plan your migration.
```
